easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -498,22 +503,20 @@ def include_defined_parameters(keyset):
                 printed = False
                 for key in group:
                     val = self._config[key][0]
-                    def_val = DEFAULT_CONFIG[key]
-                    if val not in def_val:
+                    if val != default_values[key]:
                         ebtxt.append("%s = %s" % (key, quote_str(val, escape_newline=True)))
                         printed_keys.append(key)
                         printed = True
                 if printed:
                     ebtxt.append("")
 
-
         # print easyconfig parameters ordered and in groups specified above
         ebtxt = []
         printed_keys = []
         include_defined_parameters(grouped_keys)
 
         # print other easyconfig parameters at the end
-        for key, [val, _, _] in sorted(DEFAULT_CONFIG.items()):
+        for (key, val) in default_values.items():
             if key not in printed_keys and key not in last_keys and val != self._config[key][0]:
```

Comment:
let's make this a little bit more readable (and also faster, by not using `.items()`):

```python
keys_to_ignore = printed_keys + last_keys
for key in default_values:
    if key not in keys_to_ignore and default_values[key] != self[key]:
```